mgo/find.go

```diff
@@ -272,23 +272,33 @@ func generateCursor(result interface{}, paginatedField string, shouldSecondarySo
 	if result == nil {
 		return "", fmt.Errorf("the specified result must be a non nil value")
 	}
+	// Handle pointer values and reduce number of times reflection is done on the same type.
+	var typ reflect.Type
```

Comment:
Could you please make the same changes to support pointer values and reduce the reflection call to the mgo implementation?

Reply:
You commented on the changes that are made to the mgo implementation ^^
But I can add tests ;)

Reply:
https://www.google.com/url?sa=i&url=https%3A%2F%2Fgiphy.com%2Fexplore%2Fhomer-bushes&psig=AOvVaw3nOeg7UAve8xWhFjTloy-w&ust=1594382164043000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCID5msiOwOoCFQAAAAAdAAAAABAD

Reply:
![giphy](https://user-images.githubusercontent.com/15125670/87037150-cb2c2580-c1b9-11ea-9535-23ba662839cd.gif)

Reply:
Forgo that I had made changes to both actually, that's why the description was a bit misleading.